druntime/src/core/sys/posix/config.d

```diff
@@ -88,7 +88,7 @@ else version (CRuntime_Musl)
     enum __REDIRECT          = false;
 
     // Those three are irrelevant for Musl as it always uses 64 bits off_t
-    enum __USE_FILE_OFFSET64 = _FILE_OFFSET_BITS == 64;
+    enum __USE_FILE_OFFSET64 = false;
```

Comment:
based on the comment of line 90, this should be `true` or be completely removed.